Python/import.c

```diff
@@ -2077,15 +2142,14 @@ PyImport_ImportName(PyObject *builtins, PyObject *globals, PyObject *locals,
                     PyObject *name, PyObject *fromlist, PyObject *level)  // replaces PyImport_DeferredImportName
 {
     PyThreadState *tstate = _PyThreadState_GET();
-    int verbose = _PyInterpreterState_GetConfig(tstate->interp)->verbose;
     int lazy_imports_enabled = _PyInterpreterState_GetConfig(tstate->interp)->lazy_imports;
 
     if (!lazy_imports_enabled) {
         return PyImport_EagerImportName(builtins, globals, locals, name, fromlist, level, NULL);
     }
-    if (verbose) {
-        fprintf(stderr, "# lazy import '%s'\n", PyUnicode_AsUTF8(name));
-    }
+
+    // _verbose_message will check verbose mode
+    _verbose_message("# lazy import '%s'\n", PyUnicode_AsUTF8(name));
```

Comment:
One problem with `_verbose_message()` is that it won't receive multiple arguments. This could be fixed by using variable number of arguments and `<stdarg.h>` (`...`, `va_list`, `va_start`, `va_arg` and `va_end`), but it's probably more complexity that we need.

Reply:
After talking to @Kronuz, we think it is not worth getting into too much for now. Thus, I marked that only one args can be used in `_verbose_message`. This will help to clarify its functional behavior and avoid misuse.